.travis.yml

```diff
@@ -1,17 +1,23 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
+    - 5.3
+    - 5.4
+    - 5.5
+    - 5.6
+    - hhvm
+
+env:
+    - SYMFONY_VERSION="2.3.*"
+    - SYMFONY_VERSION="2.4.*"
+    - SYMFONY_VERSION="2.5.*"
+    - SYMFONY_VERSION="2.6.*"
```

Comment:
Yeah, i'm sure we can find ways to optimize build times. This was a first commit to get feedback.